src/common/addNew.js

```diff
@@ -0,0 +1,22 @@
+import { List } from './lists.js';
+
+const addNew = (e) => {
+  e.preventDefault();
+  const description = document.getElementById('input-todo').value;
+  const list = new List();
+  list.add(description, false);
+  document.getElementById('input-todo').value = '';
+  const storedTodoList = JSON.parse(localStorage.getItem('toDolist'));
+  if (localStorage.getItem('toDolist') == null) {
+    localStorage.setItem('toDolist', JSON.stringify(list.list));
+    /* eslint-disable */
+    location.reload();
+    /* eslint-enable */
+  } else {
+    const newList = storedTodoList.concat(list.list);
+    localStorage.setItem('toDolist', JSON.stringify(newList));
+    /* eslint-disable */
+    location.reload();
+  }
+};
```

Comment:
- [ ] Kindly make sure the property `index` is set to the value of the new array length (i.e. if a 5th task is added to the list, the index of that task should equal 5) or to be more clear, the 1st `todo task` in the **todoList** should have the `index = 1`, the second one(`index = 2`),...

Reply:
Changed the same